src/spares/oc_presets

```diff
@@ -0,0 +1,220 @@
+ /*
+ Copyright (c) 2019 Romka Swallowtail
+
+ Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We prefer GPLv2. If possible can you change this?

Reply:
yes

Reply:
I did